WordPress/src/main/java/org/wordpress/android/util/AnalyticsUtils.java

```diff
@@ -218,15 +220,25 @@ public static void trackWithReaderPostDetails(AnalyticsTracker.Stat stat, long b
         trackWithReaderPostDetails(stat, ReaderPostTable.getBlogPost(blogId, postId, true));
     }
 
-    public static void trackWithBlogPostDetails(AnalyticsTracker.Stat stat, long blogId, long postId) {
+    public static void trackWithBlogPostDetails(AnalyticsTracker.Stat stat, String blogId, String postId) {
```

Comment:
Nice catch. It will prolly be OK but just to be on the safe side, I made the tracking methods take the `long`s again. So, it's only the slug related track that will use the `String`s. The relevant commit is 9449e3b. Thanks!